### ISSUE TITLE
connections

### ISSUE THREAD

**Issue by apintex** [prominic/Moonshine-IDE#304]
What kind of connections are these?

![captura de ecra 2018-06-12 as 09 44 32](https://user-images.githubusercontent.com/6188954/41280329-9afa47ae-6e26-11e8-9056-32d0ab842167.png)

**Comment by piotrzarzycki21** [comment]
@apintex This is logs from language server. Each project have it's own instance. We are using [extension](https://as3mxml.com/) from Josh to provide code completion and several other features in Moonshine.

**Comment by rat-moonshine** [comment]
I think, we probably want to add some more information against these so a regular user can understand what is happening than its tech terms.

**Comment by piotrzarzycki21** [comment]
Good idea, but I'm not sure what should we actually display there. @joshtynjala can you suggest something ?

**Comment by joshtynjala** [comment]
Those messages are going to disappear soon. When I'm done refactoring, that xsocket library won't be used anymore.
